docs/releases.md

```diff
@@ -1,6 +1,44 @@
 # Ixia-c Release Notes and Version Compatibility
 
-## Release  v0.0.1-3662 (Latest)
+## Release  v0.0.1-3698 (Latest)
+> 15th December, 2022
+
+#### Announcement
+
+`ixia-c` container images is hosted on [GitHub Container Registry](https://github.com/orgs/open-traffic-generator/packages) we stopped publishing `ixia-c` container images to [DockerHub](https://hub.docker.com/r/ixiacom).
```

Comment:
Modify :
`ixia-c` container images are hosted on [GitHub Container Registry] (https://github.com/orgs/open-traffic-generator/packages), We stopped publishing `ixia-c` container images to [DockerHub](https://hub.docker.com/r/ixiacom).'